stumpy/gpu_stump.py

```diff
@@ -1,5 +1,6 @@
 # STUMPY
-# Copyright 2019 TD Ameritrade. Released under the terms of the 3-Clause BSD license.
+# Copyright 2019 TD Ameritrade, 2020 NVIDIA CORPORATION.
```

Comment:
@gmarkall I hope this is not a big issue but I spoke with our manager of "Office of the Open Source" at TD Ameritrade and we are not able to change the header. Could we create, say, an `AUTHORS` file or something to acknowledge your contribution instead? I just want to make sure that you get what you need out of this as well.